dashboards-observability/public/components/integrations/plugins/nginx/doc/index.tsx

```diff
@@ -0,0 +1,197 @@
+            <br />
+            <EuiText data-test-subj="createPageTitle" size="s">
+              {NginxConfig.STEP3_DESC}
+            </EuiText>
+            <br />
+            <EuiCodeBlock language="json" overflowHeight={300}>
+              {NginxConfig.CONFIGURATION_STEP3}
+            </EuiCodeBlock>
+            <br />
+            <EuiText data-test-subj="createPageTitle" size="s">
+              {NginxConfig.STEP5_DESC}
+            </EuiText>
+            <br />
+            <EuiCodeBlock language="json" overflowHeight={300}>
+              {NginxConfig.CONFIGURATION_STEP4}
+            </EuiCodeBlock>
+            <br />
+            <EuiTitle data-test-subj="createPageTitle" size="s">
+              <h6>{NginxConfig.STEP_4}</h6>
+            </EuiTitle>
+            <br />
+            <EuiText data-test-subj="createPageTitle" size="s">
+              {NginxConfig.STEP4_DESC}
+            </EuiText>
+            <br />
+            <EuiCodeBlock language="json" overflowHeight={300}>
+              {NginxConfig.CONFIGURATION_STEP5}
+            </EuiCodeBlock>
+            <br />
+            <EuiLink
+              href={
+                'https://github.com/opensearch-project/observability/issues/646#issuecomment-1153889558'
+              }
+              target="_blank"
+            >
+              Click here for more details
+            </EuiLink>
+            <br />
+          </EuiPageContent>
+        }
+        flyoutFooter={
+          <EuiFlyoutFooter>
+            <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween">
+              <EuiFlexItem grow={false}>
+                <EuiButton onClick={closeIntegrationFlyout}>Close</EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlyoutFooter>
+        }
+        ariaLabel="configurationLayout"
+      />
+    );
+  }
+
+  return (
+    <div>
+      <EuiPageBody component="div" style={{ marginLeft: '28px' }}>
+        <EuiPageHeader>
+          <EuiPageHeaderSection>
+            <EuiTitle data-test-subj="createPageTitle" size="l">
+              <h1>Document panel</h1>
+            </EuiTitle>
+          </EuiPageHeaderSection>
+        </EuiPageHeader>
+        <EuiPageContent id="appInfo">
+          <EuiPageContentHeader>
+            <EuiPageContentHeaderSection>
+              <EuiTitle size="m">
+                <h2>{appName} information</h2>
+              </EuiTitle>
+            </EuiPageContentHeaderSection>
+          </EuiPageContentHeader>
+          <EuiHorizontalRule />
+          <EuiTitle data-test-subj="createPageTitle" size="m">
+            <h6>Installation</h6>
+          </EuiTitle>
+          <br />
+          <EuiText grow={false}>
+            NGINX is open source software for web serving, reverse proxying, caching, load
+            balancing, media streaming, and more. It started out as a web server designed for
+            maximum performance and stability.
+          </EuiText>
+          <br />

```

Comment:
Instead of putting line breaks like this, try using containers for the specific block of text and try putting proper padding in between them to add separation. Line breaks look different (and have different amount of spacing) in different browsers. If you are putting padding-bottom, then do not put padding below the last section.